app/gateway/changelog.md

```diff
@@ -408,6 +408,68 @@ hang when attempting to expand an API.
   of the `http-log` plugin and the `endpoint` field of the `opentelemetry` plugin,
   do not accept reference values due to incorrect field validation.
 
+## 3.3.1.1
+**Release Date** 2023/10/12
+
+### Breaking Changes
+
+* **Ubuntu 18.04 support removed**: Support for running Kong Gateway on Ubuntu 18.04 ("Bionic") is now deprecated,
+as [Standard Support for Ubuntu 18.04 has ended as of June 2023](https://wiki.ubuntu.com/Releases).
+Starting with Kong Gateway 3.2.2.4, Kong is not building new Ubuntu 18.04
+images or packages, and Kong will not test package installation on Ubuntu 18.04.
+
+    If you need to install Kong Gateway on Ubuntu 18.04, substitute a previous 3.2.x 
+    patch version in the [installation instructions](/gateway/3.2.x/install/linux/ubuntu/).
+
+* Amazon Linux 2022 artifacts are renamed to Amazon Linux 2023, based on AWS's own renaming.
+* CentOS packages are now removed from the release and are no longer supported in future versions.
```

Comment:
```suggestion
* CentOS packages are discontinued in this release and will not be supported in future versions.
```

Reply:
Also leaving this one as-is because we have to say "removed". They weren't discontinued here, they were discontinued a long time ago. CentOS was an accidentally published package that shouldn't have existed, as we stopped publishing CentOS packages in 3.0 (maybe even before that).